src/main/resources/schema.sql

```diff
@@ -1 +1,26 @@
-USE alkemy;
\ No newline at end of file
+USE alkemy;
+
+CREATE TABLE roles
```

Comment:
agreguesmos IF NOT EXISTS antes del nombre de la tabla asi corre la 1era vez solamente